osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -7,24 +7,35 @@
 using osu.Framework.Graphics;
 using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Objects.Drawables;
-using osu.Game.Rulesets.Objects.Types;
 using osu.Game.Rulesets.Osu.Objects;
 using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Configuration;
+using osu.Framework.Configuration;
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects
+    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects, IReadFromConfig
     {
         public override string Description => @"Play with no approach circles and fading notes for a slight score advantage.";
         public override double ScoreMultiplier => 1.06;
-
         private const double fade_in_duration_multiplier = 0.4;
         private const double fade_out_duration_multiplier = 0.3;
+        private Bindable<bool> increaseFirstObjectVisibility;
+
+        public void ReadFromConfig(OsuConfigManager config)
+        {
+            increaseFirstObjectVisibility = config.GetBindable<bool>(OsuSetting.IncreaseFirstObjectVisibility);
+        }
 
         public void ApplyToDrawableHitObjects(IEnumerable<DrawableHitObject> drawables)
         {
             foreach (var d in drawables.OfType<DrawableOsuHitObject>())
             {
+                //Don't hide the first object if the increaseFirstObjectVisibility is true ("drawables" are in a reverse order -> Last() )
+                //If increaseFirstObjectVisibility is null then we're exectuing one of the tests for the hidden mod. Since it's too much
+                //work for just this interface to work on this tests (DI of the config..), I just make sure to not throw a null exception
+                if (d == drawables.Last() && (increaseFirstObjectVisibility == null || increaseFirstObjectVisibility)) continue;

```

Comment:
Wouldn't initialising `increaseFirstObjectVisibility` fix all problems with test and make comments like above unnecessary?
`private Bindable<bool> increaseFirstObjectVisibility = new Bindable<bool>();`?

Reply:
You could've used the "??" operator as well

Reply:
Totally forgot about this operator, thanks for the reminder :)